src/dropdownButton/index.tsx

```diff
@@ -1,111 +1,46 @@
-import React, { PureComponent } from 'react'
+import React from 'react'
 import cc from 'classcat'
+import style from './style'
 import { ChevronIcon } from 'icon'
 import { color } from '_utils/branding'
-import style from './style'
-
-const KEYCODES = {
-  ESCAPE: 27,
-}
-
-export interface DropdownButtonProps {
-  children: React.ReactNode,
-  dropdown: React.ReactNode,
-  dropdownPosition?: 'left' | 'right',
-  dropdownWithPointer?: boolean,
-  className?: Classcat.Class,
-}
 
-export interface DropdownButtonState {
-  open: boolean,
+export interface DropdownMenu {
+  onClick: (event: React.MouseEvent<HTMLElement>) => void,
+  children: JSX.Element | string
+  open?: boolean,
+  className?: string
```

Comment:
Why not use `Classcat.Class` type?

Reply:
As discussed IRL, it can be used on consumer side because we don't use `cc` in this component